internal/server/models/destination.go

```diff
@@ -24,9 +24,7 @@ type Destination struct {
 
 func (d *Destination) ToAPI() *api.Destination {
 	connected := false
-	// TODO: this should be configurable
-	// https://github.com/infrahq/infra/issues/2505
```

Comment:
I commented on this issue that with long-polling we can't realistically make this configurable without making other changes to how we track LastSeenAt.